test/test-v2-to-v3.js

```diff
@@ -0,0 +1,85 @@
+import fs from 'fs';
+import path from 'path';
+import { compareFormSchemas } from '../src/fulcrum-schema';
+import chai from 'chai';
+
+chai.should();
+
+let oldForm = null;
+let v2Form = null;
+let v3Form = null;
+
+const fixture = (file) => fs.readFileSync(path.join('test', 'fixtures', file)).toString();
+
+beforeEach(() => {
+  oldForm = JSON.parse(fs.readFileSync(path.join('test', 'fixtures', 'form.json'))).form;
```

Comment:
would it be worth renaming this to v1Form and renaming the fixtures accordingly (instead of form and form-new)?

Reply:
Yeah, that makes sense.